components/modal_controller/index.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {closeModal} from '../../actions/views/modals';
```

Comment:
Can you change this to use an absolute path? We prefer to use those except in cases like below where you're importing the component into index.js

Reply:
No problem, I just pushed the change.